samples/nrf9160/modem_shell/sample.yaml

```diff
@@ -142,27 +159,30 @@ tests:
                 DTC_OVERLAY_FILE="nrf9160dk_with_nrf7002ek.overlay"
     integration_platforms:
       - nrf9160dk_nrf9160_ns
+      - nrf9161dk_nrf9161_ns
     platform_allow: nrf9160dk_nrf9160_ns nrf9161dk_nrf9161_ns
     tags: ci_build
   sample.nrf9160.modem_shell.ppp:
     build_only: true
     extra_args: OVERLAY_CONFIG=overlay-ppp.conf DTC_OVERLAY_FILE="ppp.overlay"
     integration_platforms:
       - nrf9160dk_nrf9160_ns
+      - nrf9161dk_nrf9161_ns
     platform_allow: nrf9160dk_nrf9160_ns nrf9161dk_nrf9161_ns
     tags: ci_build
   sample.nrf9160.modem_shell.bt:
     build_only: true
     extra_args: OVERLAY_CONFIG=overlay-bt.conf DTC_OVERLAY_FILE="bt.overlay"
     integration_platforms:
       - nrf9160dk_nrf9160_ns
-    platform_allow: nrf9160dk_nrf9160_ns nrf9161dk_nrf9161_ns
+    platform_allow: nrf9160dk_nrf9160_ns
```

Comment:
Is this correct, it differs from all the others?

Reply:
Yes, the nRF9161DK does not have a bluetooth chip.